src/openforms/prefill/contrib/family_members/typing.py

```diff
@@ -0,0 +1,16 @@
+from typing import Literal, TypedDict
+
+from .constants import FamilyMembersTypeChoices
+
+
+class FamilyMembersPartnerOptions(TypedDict):
+    variable_key: str
+    type: Literal[FamilyMembersTypeChoices.partners, FamilyMembersTypeChoices.children]
```

Comment:
you can't do `Literal` with dynamic code, it has to be statically defined :(

I'd go for `type: FamilyMembersTypeChoices` here